### DISCUSSION TITLE
Reorder fields in "Bug Report" issue template

### DISCUSSION THREAD

**Discussion by makisukurisu** [pear-devs/pear-desktop#1782] [Ideas]
Currently, the New Issue for a bug report looks like this:
![image](https://github.com/th-ch/youtube-music/assets/49622129/e52d3373-b4a4-4be9-b6c5-53432051787e)

With `Reproduction steps` section before `Expected Behavior` and `Actual Behavior`, I believe, it's best to change the ordering of those sections to:
- `Expected Behavior`
- `Actual Behavior`
- `Reproduction steps`
Or:
- `Actual Behavior`
- `Reproduction steps`
- `Expected Behavior`

In my opinion, having reproduction steps after describing the issue itself is a little easier, since you will necessarily mention some steps when describing the behaviour, and thus — can be more accurate in the reproduction steps section.

This is mainly based on my personal preferences and experience from other bug reporting platforms, for example:
- [Telegram's bug tracker](bugs.telegram.org) (bugs template)
- [ytmdesktop's](https://github.com/ytmdesktop/ytmdesktop) bug report template
- [Unigram issues template](https://github.com/UnigramDev/Unigram/issues/3010) (outdated)

If you agree with this, perhaps, we can vote to change the ordering of fields in the template.
If you have better ideas, please let us know in the discussion comments.

**Comment by JellyBrick** [comment]
I agree with the former

- Expected Behavior
- Actual Behavior
- Reproduction steps

e.g. `Electron`

![image](https://github.com/th-ch/youtube-music/assets/16558115/a31ab84e-0801-40ce-8ba0-cf3659b3178a)
